fees-rewards.md

```diff
@@ -0,0 +1,75 @@
+
+An "expected monthly node operational costs" variable is set through the governance protocol, which is designed to reflect how much a node operator would pay for hardware, hosting etc, in a month. This is analogous to the proof of work difficulty found in other blockchains. This effects demand for running a node.
+
+Aggregator nodes incur cost from submitting transactions to L1, through the following:
+
+- Storing rollup data on Ethereum.
+- The overhead of processing the L1 transaction that contains the rollup, e.g. signing transactions.
+
+## Fee per transaction
+
+The total fee a user must pay has three components.
+
+1. A proportion of the monthly operational cost for all nodes. The proportion is calculated based on the computation of the current transaction as a proportion of the moving average of all L2 transactions over the month. Represented as:
+
+![proportion_equation.png](images/equation_1.png)
+
+![operation_equation.png](images/equation_2.png)
+
+2. The cost incurred by nodes for L1 storage, represented as:
+
+![storage_equation.png](images/equation_3.png)
+
+3. A proportion of the additional overhead cost incurred by nodes for processing rollups is as follows:
+
+The idea behind this is:
+
+- At the point of transaction, it is impossible to know what the true cost of the future rollup is
+- Users need to pay their fair share of what this future rollup cost will be
+- The best indicator of what the future rollup cost might be is a moving average of the previous n rollups
+- Where n is high, the averaging is performed over a larger number of rollups and therefore less volatile and vice versa. High n adds more risk to Aggregators
+- L2 gas can be computed as Obscuro has the same metering mechanism as L1. As a result of this metering, we can work out the gas consumed and directly map this to the L1 moving average gas cost
+
+![overhead_equation_4.png](images/equation_4.png)
+
+This gives the expected fee per transaction as:
+
+![total_fee_equation_5.png](images/equation_5.png)
+
+##
+
+## Rewards for L2 nodes
+
+The Obscuro protocol rewards both Validators and Aggregators that monitor the network via a lottery. The lottery randomly rewards active nodes each round with an OBX reward for participating. Nodes prove they are alive by claiming their reward within the subsequent few blocks; else, the reward is burned. The reward in each rollup is calculated to cover the operational costs of these nodes.
+
+Aggregator nodes have an additional reward mechanism that pays out rewards in OBX when they successfully win the right to validate and publish a rollup. This reward covers at least the L1 gas costs incurred.
+
+The lottery pay out amount per rollup round is represented as:
+
+![active_node_reward_equation_6.png](images/equation_6.png)
+
+## Bootstrapping the network
+
+##
+
+When the network first spins up, it is prudent to expect low transaction volume. Low transaction volume creates a situation where:
```

Comment:
When the network first spins up, it is prudent to expect low transaction volume, which can create a situation where: